src/yaml.ts

```diff
@@ -33,6 +35,9 @@ const schemaTypes: jsyaml.Type[] = [];
 export const customTags: {[key: string]: typeof Tag} = {};
 export const cfnIntrinsicTags: {[key: string]: typeof Tag} = {};
 
+export const isCustomIidyTag = (tag: Tag): boolean => !!customTags[tag.tag_name];
+export const isCfnIntrinsicTag = (tag: Tag): boolean => !!cfnIntrinsicTags[tag.tag_name];
```

Comment:
this is just a personal preference. These days I find explicit casting to be more readable:

```suggestion
export const isCfnIntrinsicTag = (tag: Tag): boolean => Bool(eancfnIntrinsicTags[tag.tag_name]);
```

There's potential for small performance loss, but I'm assuming this isn't something that needs to be highly optimized anyway.